doc/nrf/includes/vsc_build_and_run_nrf53.txt

```diff
@@ -0,0 +1,14 @@
+1. Open Visual Studio code.
+#. Click on the :guilabel:`nRF Connect for VS Code` icon in the left navigation bar and then click :guilabel:`Open`.
```

Comment:
```suggestion
#. Click on the :guilabel:`nRF Connect for VS Code` icon in the left navigation bar.
```
I think it is enough to click on the `nRF Connect` icon. I don't see `Open` button.